src/_main_/settings.py

```diff
@@ -18,6 +18,12 @@
 import sentry_sdk
 from sentry_sdk.integrations.django import DjangoIntegration
 
+
+DJANGO_ENV = os.environ.get("DJANGO_ENV","remote")
+DEV_SERVER = False
```

Comment:
can we use a name like `RUN_SERVER_LOCALLY`
Also can we add comments to describe what each of these stand for?

Reply:
Yes, done